test/languages.ts

```diff
@@ -475,7 +475,10 @@ export const SwiftLanguage: Language = {
     { density: "normal" },
     { "url-session": "true" },
     { "access-level": "internal" },
-    { "access-level": "public" }
+    { "access-level": "public" },
+    { "protocol": "none" },
```

Comment:
The default doesn't need a quick-test case - it will be tested on the whole test suite anyway.